services/host_service.js

```diff
@@ -0,0 +1,57 @@
+
+const {Host} = require('../db/host')
+const {TimeSlot} = require('../db/time_slot')
+const {RestUtils} = require('../utils/rest_utils')
+const {Checker} = require('./data_checker')
+
+Host.timeSlots = Host.hasMany(TimeSlot);
+
+const HostService = {
+    createHost: async () => {
+        return Host.create({
+            email: "andrzej@duda.pl",
```

Comment:
yyy to w przyszłości będzie miało jakieś paramerty, nie? A nie będziemy mieli tylko hosta Andrzeja Dudę, bo on podpisywać to spoko, ale nie wiem, czy chciałbym, żeby mi rozmowę umawiał XD

Reply:
tego w ogóle nie będzie, nie ma narazie takiej potrzeby, dodałem przez przypadek a używałem do testów XD